lib/elixometer.ex

```diff
@@ -281,6 +281,9 @@ defmodule Elixometer do
   @doc """
   Updates a counter metric. If the metric doesn't exist, the metric is created
   and the metric is subscribed to the default reporter.
+
+  If the value of the `:reset_seconds` option is greater than zero, the counter will be reset
+  automatically at the specified interval.
   """
   def update_counter(name, delta, opts \\ [reset_seconds: 60]) when is_bitstring(name) do
```

Comment:
`nil` for no reset by default sounds good to me. Would you still like the edge case of 'reset < 1 second' handled in a certain way?